ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -278,6 +282,22 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
+- as: aws-ipi-ovn-hypershift-control-plane-p1-f7
+  cron: 15 19 4,11,18,26 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      SKIP_HYPERSHIFT_PULL_SECRET_UPDATE: "true"
+      TEST_FILTERS: ~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.13 and @aws-ipi and @hypershift-hosted
```

Comment:
Due to the design of the upgrade tests, the check need to match the prepare.

Reply:
Ah,  I didn't remove UPGRADE_PRE_RUN_TAGS
```
UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.12 and @aws-ipi and @hypershift-hosted
        and not @fips'
```

Reply:
thanks, Could you help to review again?

Reply:
@liangxia thanks, Could you help to review again?